### DISCUSSION TITLE
金山文档无法查看，建议更换为腾讯文档和语雀进行分享

### DISCUSSION THREAD

**Discussion by fanvanf** [WeNeedHome/SummaryOfLoanSuspension#157] [General]
金山文档无法查看，建议更换为腾讯文档和语雀进行分享

**Comment by shawn-yee** [comment]
国产文档都不建议，封杀链接迟早的事

**Comment by pacoblack** [comment]
一纸文件过去，哪个敢不下架，前两天金山删用户文件可见一斑

**Comment by qq912276337** [comment]
google文档呢

**Reply by CHENQOK** [reply]
https://docs.google.com/spreadsheets/d/1AJu8Lg44UywYlYcEra_wWl3G2mZjBeivpRxzWSZ5_x0/edit#gid=0
